npc/re/jobs/novice/academy.txt

```diff
@@ -13,75 +13,241 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
```

Comment:
Further testing:
The full room(left,start room) is covered. #2274 might be a topic again here.
In the right room the OnTouch area of the warp to the left room also triggers navigation(confirmed range to include 52,30).